<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceManageWindow</class>
 <widget class="QMainWindow" name="DeviceManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
border-image: url(:/image/function/above-atmosphere-clouds-37728.jpg);
}</string>
   </property>
   <widget class="QWidget" name="ledwidget1" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ledpushButton1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ledpushButton2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="ledwidget2" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closeled_ico.png);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fanpushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>450</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="fanwidget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closefan_ico.png);</string>
    </property>
   </widget>
   <widget class="QWidget" name="doorwidget" native="true">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>370</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/closedoor_ico.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doorpushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>460</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/room/OFF.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>273</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/login/farsight.png);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>灯一</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>灯二</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>风扇</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>门</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
